src/Components/DrillPageMinimal.js

```diff
@@ -112,7 +121,7 @@ const styles = StyleSheet.create({
     marginLeft: '10%',
     marginTop: 15,
     marginBottom: 5,
-    width: '80%',
```

Comment:
I think the 80% was here to handle long titles

For example:

![Capture du 2020-11-12 18-05-22](https://user-images.githubusercontent.com/3810795/98971580-a81bd480-2511-11eb-9bb4-d41176793999.png)


Some people have smaller phones than we do, and they might have the problem even if we don't have it